rosserial_client/src/ros_lib/ros/node_handle.h

```diff
@@ -116,7 +116,11 @@ namespace ros {
        * Setup Functions
        */
     public:
+#ifdef ROS_MBED_HARDWARE_H_ 
+      NodeHandle_(PinName tx = USBTX, PinName rx = USBRX) : hardware_(tx, rx), configured_(false) {
+#else
       NodeHandle_() : configured_(false) {
+#endif
```

Comment:
I don't care for modifying `rosserial_client` in a device-specific way like this. Can you put the logic in the MbedHardware class instead, perhaps as a method which can be called to set it up in the desired manner even after it has been constructed inside RosserialClient?

cf. https://github.com/ros-drivers/rosserial/blob/jade-devel/rosserial_mbed/src/ros_lib/MbedHardware.h#L17